### ISSUE TITLE
Timer not starting from time set as 'duration'

### ISSUE THREAD

**Issue by dlopc** [home-assistant/frontend#23147]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

Creating a new timer with 00:01:20 (or any other value) and 'restore' set to 'enabled' ... do not start at 1:20, but 2:19 or 2:20

### Describe the behavior you expected

timer to display duration set

### Steps to reproduce the issue

1.Settings > devices > helpers > create helper
2. Scroll down to 'timer', and click
3. Give it a name, and set duration to 00:01:20
4. Enable 'restore'
5. Click on 'create'
6.  from the list of helpers , click on the recently added timer. It will show the name of the timer and 'X seconds/minutes ago' (increasing while typing this line in github). As well as 'iddle' on the right
7. Click on 'start', and 'idle' will change to regressive counter starting from 2:20
8.  One more symptom: the duration for a new timer is 00:00:00 by default, and insert/overwrite is in 'insert' always. If set to 00:01:20, saved and 'edited', the time shows as 0:01:20 (hours with just 1 digit)
...

### What version of Home Assistant Core has the issue?

2024.11.3

### In which browser are you experiencing the issue with?

Firefox 132.0.2 64bits

### Which operating system are you using to run this browser?

W10

**Comment by karwosts** [comment]
This is likely a symptom of the clock of your frontend and backend being out of sync. Check the clock on both and you may find that one is running ~60s ahead of the other. 

The timer will run for 1:20, but the frontend will display the incorrect amount of time remaining because one of the system clocks are wrong.

**Comment by dlopc** [comment]
Thanks. You got it **solved**.

But I still don't know _**how to prevent this**_ from happening again, and what to look for in the log.

This is what I did:

- reboot the router, 
- reboot Home-assistant into safemode and back to normal, 
- reboot the server where home-asssitant is running on docker, 
- and wait for some 30 minutes. 

Now it looks good both on the docker instance and on y old Rpi3 with earlier relase...I am migrating from. The error was the same on both systems before everyting was restarted. I noticed the one minute gap between my W10 and the rest (Debian, HA, Android phone). I will setup my Debian server to have a static IP and NTP client (my router only allows reserved IP within the DHCP scope)

Thanks again
